packages/app/src/navigation/SidebarNavigation.cy.tsx

```diff
@@ -86,4 +116,74 @@ describe('SidebarNavigation', () => {
     mountComponent()
     cy.findByTestId('sidebar-link-specs-page').should('have.class', 'router-link-exact-active')
   })
+
+  context('debug status badge', () => {
+    it('renders new badge without cloudProject', { viewportWidth: 1280 }, () => {
+      cy.clock(new Date('December 20, 2022'))
+
+      mountComponent()
+
+      cy.findByLabelText('New Debug feature').should('be.visible').contains('New')
+      cy.percySnapshot('Debug Badge:collapsed')
+
+      cy.findByLabelText(defaultMessages.sidebar.toggleLabel.collapsed, {
+        selector: 'button',
+      }).click()
+
+      cy.percySnapshot('Debug Badge:expanded badge')
+    })
+
+    it('renders new badge when run status is "NOTESTS" or "RUNNING"', () => {
+      cy.clock(new Date('December 20, 2022'))
+
+      for (const status of ['NOTESTS', 'RUNNING'] as CloudRunStatus[]) {
+        mountComponent({ cloudProject: { status, numFailedTests: 0 } })
+        cy.findByLabelText('New Debug feature').should('be.visible').contains('New')
+      }
+    })
+
+    it('renders no badge if no cloudProject and released > 2 months ago', () => {
+      // Set to February 15, 2023 to see this fail
+      cy.clock(new Date('February 20, 2023'))
```

Comment:
Use date addition to calculate this date instead of hardcoding a date to show the relationship to the "release" date.

Reply:
Addressed in [4c21f5f](https://github.com/cypress-io/cypress/pull/25152/commits/4c21f5f4d005a19a9dbe52ffac3101f9bfeb513f)